### PR TITLE
Test with Django 1.11

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ env:
   - DJANGO="Django>=1.8,<1.9"
   - DJANGO="Django>=1.9,<1.10"
   - DJANGO="Django>=1.10,<1.11"
+  - DJANGO="Django>=1.11a,<2.0"
```
Comment (felixxm) [inline .travis.yml]: Small typo `1.11a` -> `1.11a1`

File: tox.ini [hunk 2]
```diff
@@ -18,3 +18,4 @@ deps =
     dj18: Django>=1.8,<1.9
     dj19: Django>=1.9,<1.10
     dj110: Django>=1.10,<1.11
+    dj111: Django>=1.11a,<2.0
```
Comment (felixxm) [inline tox.ini]: Small typo `1.11a` -> `1.11a1`

File: .travis.yml [hunk 3]
```diff
@@ -28,6 +29,12 @@ matrix:
       python: "3.2"
     - env: DJANGO="Django>=1.10,<1.11"
       python: "3.3"
+    - env: DJANGO="Django>=1.11a,<2.0"
+      python: "3.2"
+    - env: DJANGO="Django>=1.11a,<2.0"
+      python: "3.3"
+    - env: DJANGO="https://github.com/django/django/archive/master.tar.gz"
+      python: "2.7"
```
Comment (felixxm) [inline .travis.yml]: Django 2.0 doesn't support `pypy`. Maybe you can add it to exclude matrix in this PR.

File: .travis.yml (hunk 1 above)
Comment (claudep) [inline .travis.yml]: Looks like it depends on the pip version… Thanks for the fix.

File: .travis.yml (hunk 3 above)
Comment (claudep) [inline .travis.yml]: I don't know enough about pypy to change anything related. I noticed though there is a pypy3 option to use on Travis.

**Comment by craigds** [comment]
Yeah, I'm happy to remove `pypy` altogether. I doubt many people use it. In any case it behaves pretty much the same as CPython 2.7 from mptt's perspective, so no need to have it tested separately.

**Comment by craigds** [comment]
but this change is good without that anyway. thanks!

File: .travis.yml (hunk 1 above)
Comment (claudep) [inline .travis.yml]: Despite the correction, it continues to fail (sometimes?).
https://travis-ci.org/django-mptt/django-mptt/jobs/203252723

Comment (craigds) [inline .travis.yml]: super strange. Latest build worked. I'm hoping it stays that way. Possibly travis had cached something oddly. ¯\_(ツ)_/¯

**Pull request merged**